roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -317,6 +317,8 @@ spec:
               if [ ! -f /etc/receptor/receptor.conf ]; then
                 cp /etc/receptor/receptor-default.conf /etc/receptor/receptor.conf
                 sed -i "s/HOSTNAME/$HOSTNAME/g" /etc/receptor/receptor.conf
+                chown awx /etc/receptor/receptor.conf
+                chmod 664 /etc/receptor/receptor.conf
```

Comment:
We should actually do this in the Dockerfile instead, the cat the receptor.conf into the "stub" receptor.conf file, like we do for rsyslog.conf.

Reply:
at least we can't do chown since awx is not a user that exist

Reply:
i think just the chmod will work but i cant reproduce this issue on my openshift cluster

Reply:
/etc/receptor is a emptyDir mount so when task container and web container mount the emptyDir it should be own by them so ownership shouldnt be a issue

Reply:
The Pod Security Context supports setting an fsGroup, which allows you to set the group ID that owns the volume, and thus who can write to it.

[More](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)